proto/rill/local/v1/api.proto

```diff
@@ -122,6 +98,7 @@ message GetCurrentUserRequest {}
 message GetCurrentUserResponse {
   admin.v1.User user = 1;
   repeated string rill_user_orgs = 2;
+  string assumed_user_email = 3;
```

Comment:
Would suggest `bool is_representing_user` because:
1. We use the word "represent" not "assume" internally, so this is more consistent
2. The value is the same as `user.email`, so better to make it a bool to avoid ambiguity